apps/mobile/src/features/ImportPrivateKey/ImportPrivateKey.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { render, screen, fireEvent, act, waitFor } from '@/src/tests/test-utils'
+import { ImportPrivateKey } from './ImportPrivateKey.container'
+
+jest.mock('@/src/navigation/useScrollableHeader', () => ({
+  useScrollableHeader: () => ({
+    handleScroll: jest.fn(),
+  }),
+}))
```

Comment:
maybe we should move this to a global mock. We have a lot of screens with this hook now.